x/qgb/orchestrator/orchestrator_client.go

```diff
@@ -164,11 +228,89 @@ func (oc *orchestratorClient) SubscribeDataCommitment(ctx context.Context) (<-ch
 					End:        endHeight,
 					Nonce:      nonce,
 				}
-
 			}
 		}
-
 	}()
 
 	return dataCommitments, nil
 }
+
+func (oc *orchestratorClient) addOldDCAttestations(
+	ctx context.Context,
+	dataCommitmentsChan chan ExtendedDataCommitment,
+) error {
```

Comment:
if we're not going to handle this error, and only log, then I don't think we should be returning any

Reply:
I think we will handle this error with the new design. But I can remove it, I'm okey with both.
Should I ?

Reply:
yeah, lets get rid of it for now. We'll probably need to change this function a bit then anyway

Reply:
done